File: abstractos 28-08-2025/abstractos.js
class Mascota {
    constructor(nombre, tipo, edad, raza) {
        this.nombre = nombre;
        this.tipo = tipo;
        this.edad = edad;
        this.raza = raza;
    }
}
const cat = new Mascota("Firulais", "Perro", 3, "Labrador");
console.log(cat);

File: abstractos 28-08-2025/Abstractos.cs
using System;

public class Mascota
{
    public string Nombre { get; set; }
    public string Tipo { get; set; }
    public int Edad { get; set; }
    public string Raza { get; set; }
}
public class Program
{
    public static void Main(string[] args)
    {
         Mascota miMascota = new Mascota 
        {
            Nombre = "Fido",
            Tipo = "Perro",
            Edad = 5,
            Raza = "Labrador"
        };
        Console.WriteLine($"Nombre: {miMascota.Nombre}, Tipo: {miMascota.Tipo}, Edad: {miMascota.Edad}, Raza: {miMascota.Raza}");
    }
}